plugins/apitracing/src/lib/FunctionHook.cpp

```diff
@@ -55,24 +56,59 @@ namespace ApiTracing
             }
 
             auto extractedParameters = extractor->extractParameters(event, parameterInformation);
-            logParameterList(extractedParameters);
+
+            Json::Value root;
+            root["functionName"] = functionName;
+            auto output = getParameterListAsJson(extractedParameters, root);
+            auto json = output.toStyledString();
```

Comment:
Adding whitespace chars adds unnecessary bloat. The logs will become very large anyways, no need to bloat them further. If you need to make the logs more readable afterwards, you can always use tools to do so (e.g. jq)

Reply:
The toStyledString conversion is a relic from the last presentation.